packages/plugins/src/hardhat/simulate.ts

```diff
@@ -120,11 +120,11 @@ export const simulate = async (
     //    don't hardcode it, Hardhat uses the default hardfork, which is the behavior we want.
 
     // Fast forward the block number. This is necessary to prevent the following edge case:
-    // 1. We deploy the Gnosis Safe and Sphinx contracts.
+    // 1. We execute some transactions on the network.
     // 2. The simulation uses a block that's multiple confirmations behind the latest block. This is
     //    Hardhat's default behavior, which is meant to protect against chain reorgs on forks of
     //    live networks.
-    // 3. The simulation fails because some of the contracts deployed in step 1 don't exist on the
+    // 3. The simulation fails because the transactions executed in step 1 don't exist on the
```

Comment:
Changed this wording slightly because the provided example isn't relevant anymore. (We now deploy the system contracts in the simulation instead of outside the simulation, so step 1 isn't accurate anymore)

Reply:
Can we just remove this entirely? I thought the main reason we had it was because the contracts were being deployed prior to the simulation.

Reply:
I think we need to keep it. I updated the description, which should hopefully clarify how this could happen. Copying and pasting it here:
```
    // Fast forward the block number. This is necessary to prevent the following edge case:
    // 1. Some transactions are executed on the local network. These transactions could either be
    //    sent by the Sphinx team (during testing) or by the user during local development.
    // 2. The Deploy CLI command is executed, which creates a simulation using a block that's
    //    multiple confirmations behind the latest block. This is Hardhat's default behavior, which
    //    is meant to protect against chain reorgs on forks of live networks.
    // 3. The simulation fails because the transactions executed in step 1 don't exist on the
    //    Hardhat fork.
```